src/main/python/scripts/remove_duplicate_links.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import shapefile
+
+if len(sys.argv) != 3:
+    print("python remove_duplicate_links.py <input shapefile> <output location>")
+    exit()
+
+reader = shapefile.Reader(sys.argv[1])
+writer = shapefile.Writer(sys.argv[2])
+
+# START
+# accumulate adjacent links max_speed
+inbound_links = dict()
+outbound_links = dict()
+
+def adjacent_links(d, key, value):
```

Comment:
If I understood the function correctly, let's give variables sensible names:
d -> links_dict
key -> node_id
value -> link_speed

Reply:
Ah, sure. I'll update the PR. Thanks for reviewing the PR, @nikolayilyin .

Reply:
Done.